oap-ape/README.md

```diff
@@ -96,7 +96,27 @@ Extra configuration to enable ape features:
 | `spark.sql.parquet.filterPushdown ` | whether enable filter push down to Parquet native | true |
 | `spark.sql.ape.aggregation.pushdown.enabled` | (Experimental) whether enable aggregation push down to Parquet native | false | 
 |`spark.sql.ape.parquet.cache.enabled` | whether enable parquet cache, you need to start plasma service on worker nodes if you enable this | false |
-|
+|`spark.sql.ape.reader.location remote` | enable remote reader in disaggregated mode | local |
```

Comment:
option is `local` and `remote` ? Please make it more clear.

Reply:
default should be local. Updated.